### PR TITLE
Fix tasks table sorting in the UI

### PR DESCRIPTION
**Pull request by AlekseiS**
Currently sorting tasks by ID ignores custom sort function and works as if IDs are strings.
Fix several involved issues including missing 'removeQueryId()' function, missing 'var' declaration,
incorrect sort implementation for arrays.

### CODE REVIEW

**Comment by raghavsethi** [comment]
Get rid of the newline, otherwise looks good.

File: presto-main/src/main/resources/webapp/assets/query.js [hunk 1]
```diff
@@ -124,9 +123,28 @@ var TaskList = React.createClass({
...
+                    'rowsSec',
+                    'bytes',
+                    'bytesSec',
+                    'elapsedTime',
+                    'cpuTime',
+                    'bufferedBytes',
+                ]}
+                defaultSort={ {column: 'id', direction: 'asc'} }>
                 <Thead>
-                        <Th column="id" sortFunction={this.compareTaskId}>ID</Th>
```
Comment (raghavsethi) [inline presto-main/src/main/resources/webapp/assets/query.js]: It's also not clear that moving this above makes the code easier to read. I would revert this also.

Comment (AlekseiS) [inline presto-main/src/main/resources/webapp/assets/query.js]: The way it was done is not mentioned in the official documentation for Reactable and didn't work in my tests. After my change it works.
I'll remove the newline.

Comment (raghavsethi) [inline presto-main/src/main/resources/webapp/assets/query.js]: Sounds good.

**Pull request closed without merging**